src/client/pages/CheckoutDetails/components/Label.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+
+const { gray500 } = colorsV3
+
+export const Label = styled.div`
```

Comment:
Unless these are meant to be reusable I don't see a big reason to keep them in individual modules, or what do you think?

Reply:
These will be reused in at least one other component, but I don't know if that's enough to keep them separate? Maybe three's a charm?

Reply:
Yeah I usually tend to think that way -- especially for very simple components